<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyConstraintStudentsMaxBuildingChangesPerDayForm_template</class>
 <widget class="QDialog" name="ModifyConstraintStudentsMaxBuildingChangesPerDayForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="This is the title of the dialog to modify a constraint of this type">Modify students max building changes per day</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="weightTextLabel">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Weight percentage
(necessary: 100%)</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="weightLineEdit">
       <property name="text">
        <string notr="true">100</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel">
       <property name="text">
        <string>Max building changes per day</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxChangesSpinBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
